client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -417,7 +423,8 @@ export class DomainDetailsForm extends PureComponent {
 	}
 
 	focusFirstError() {
-		this.refs[ kebabCase( head( map( formState.getInvalidFields( this.state.form ), 'name' ) ) ) ].focus();
+		const firstErrorName = kebabCase( head( formState.getInvalidFields( this.state.form ) ).name );
+		( this.inputRefs[ firstErrorName ] || this.refs[ firstErrorName ] ).focus();
```

Comment:
Personal pref again. I'd expand another line or two here if I had to in favor of readability. Maybe..?

```
const ref = this.inputRefs[ firstErrorName ] || this.refs[ firstErrorName ];
ref.focus();
```

It's not impossible to follow as you have it of course but if the performance different is zero to infinitesimally small off zero I favor expanding for humans who come later.